src/main/java/com/tstepnik/planner/domain/task/TaskMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.tstepnik.planner.domain.task;
+
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface TaskMapper {
+
+    TaskDTO toDto(Task task);
+
+    Task toTask(TaskDTO taskDto);
+
+    List<TaskDTO> toDto(List<Task> tasks);
+
+    List<Task> toTasks(List<TaskDTO> tasksDto);
```

Comment:
```suggestion
    List<Task> toEntity(List<TaskDTO> tasksDto);
```

Reply:
You can create interface `EntityMapper` that will be generic and will have 4 methods. `toDto` and `toEntity` and then you can just extend `EntityMapper` in any other mappers. No duplicated code :)

Reply:
Good idea, thanks.